source/options.css

```diff
@@ -11,6 +11,13 @@ textarea {
 	-moz-tab-size: 4 !important;
 	tab-size: 4 !important;
 }
+input[type='password'] {
+	display: block;
+	width: 100%;
+	font-family: SFMono-Regular, Consolas, 'Liberation Mono', Menlo, Courier, monospace;
```

Comment:
Many things in common with `textarea` here. The shared styles should be combined, and `input[type='password']` => `input`.